src/promptflow-devkit/promptflow/_sdk/_tracing.py

```diff
@@ -101,14 +100,14 @@ def _print_tracing_url_from_local(
         url += f"?#run={run}"
     elif exp is not None:
         url += f"?#experiment={exp}"
-    elif session_id is not None:
-        url += f"?#session={session_id}"
+    elif collection is not None:
+        url += f"?#session={collection}"
```

Comment:
session=?

Reply:
good catch, that's a legacy url, will fix that after UX be compatible with this.